README.rst

```diff
@@ -1,662 +1,5 @@
-django-netjsongraph
-===================
+**WARNING**: The development of this project has moved to `openwisp-network-topology <https://github.com/openwisp/openwisp-network-topology>`_, we advise all users of django-netjsongraph  to `follow the given tutorial in openwisp-network-topology to migrate the existing database <https://github.com/openwisp/openwisp-network-topology#migrating-from-django-netjsongraph>`_.
```

Comment:
I think it would be better to move the tutorial here, so we keep the README of openwisp-network-topology leaner which will be better for most users. Users migrating from django-netjsongraph will need it only once. So it makes sense to avoid cluttering the documentation with instructions that will not be used often.

Reply:
Hmm, that's what I did earlier but changed because I thought after archiving the repository making any changes here would be difficult, but in the name of avoiding cluttering the documentation on network-topology, I agree let's move it here! :smile: